src/annotator/config.js

```diff
@@ -47,7 +47,7 @@ function configFrom(window_) {
   //
   // In environments where the config has not been injected into the DOM,
   // we try to retrieve it from the URL here.
-  var directLinkedID = annotationQuery.extractAnnotationQuery(window_.location.href);
+  var directLinkedID = extractAnnotationQuery(window_.location.href);
```

Comment:
I'm gonna look into getting rid of it anyway. I think it either needs to return just a string (and the code that uses it can figure out whether it's an annotation ID or a query), or it needs to be two separate settings with two separate setting functions